internal/app/app_darwin.go

```diff
@@ -0,0 +1,188 @@
+package app
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/assets"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/installation"
+	"github.com/ActiveState/cli/internal/strutils"
+)
+
+const (
+	execFileSource   = "exec.sh.tpl"
+	launchFileSource = "com.activestate.platform.app.plist.tpl"
+	iconFile         = "icon.icns"
+	assetAppDir      = "placeholder.app"
+)
+
+func (a *App) install() error {
```

Comment:
This still has too much logic for my taste, I responded to your comment about using go:embed, I still think that's the way to go.